src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { UbsAdminTariffsAddServicePopupComponent } from './ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component';
+import { UbsAdminTariffsDeletePopupComponent } from './ubs-admin-tariffs-delete-popup/ubs-admin-tariffs-delete-popup.component';
+
+@Component({
+  selector: 'app-ubs-admin-tariffs',
+  templateUrl: './ubs-admin-tariffs.component.html',
+  styleUrls: ['./ubs-admin-tariffs.component.scss']
+})
+export class UbsAdminTariffsComponent implements OnInit {
+  public icons = {
+    edit: './assets/img/profile/icons/edit.svg',
+    delete: './assets/img/profile/icons/delete.svg'
+  };
+  constructor(public dialog: MatDialog) {}
+  ngOnInit(): void {}
+
```

Comment:
empty hook

Reply:
should i delete it if in the near future this hook won't be empty?

Reply:
Yes, you will add it in the future))

Reply:
done)